src/Buerokratt.Common/CentOps/Extensions/LoggerExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace Buerokratt.Common.CentOps.Extensions
+{
+    public static class LoggerExtensions
+    {
+        private static readonly Action<ILogger, DateTime, Exception?> refreshParticipantCache =
+            LoggerMessage.Define<DateTime>(
+                LogLevel.Information,
+                new EventId(4, nameof(RefreshingParticipantCache)),
+                "Refreshing participant cache at '{Time}'");
+
+        private static readonly Action<ILogger, int, int, Exception?> participantCacheRefreshed =
+            LoggerMessage.Define<int, int>(
+                LogLevel.Information,
+                new EventId(5, nameof(ParticipantCacheRefreshed)),
+                "Participant cache refreshed. '{Added}' added or updated. '{Removed}' removed.");
+
+        private static readonly Action<ILogger, Exception?> participantCacheRefreshFailed =
+            LoggerMessage.Define(
+                 LogLevel.Critical,
+                   new EventId(6, nameof(ParticipantCacheRefreshFailure)),
+                   "Participant Cache Refresh Failed!");
```

Comment:
To keep the messaging consistent with the other logs:

```suggestion
                   "Participant cache refresh failed!");
```